install/k8s/install.sh

```diff
@@ -251,13 +257,18 @@ rm -f /usr/bin/netctl
 cp ./netctl /usr/bin/
 # Install Contiv
 $kubectl apply -f $contiv_yaml
-if [ "$fwd_mode" = "routing" ]; then
-	sleep 60
-	netctl --netmaster http://$netmaster:9999 global set --fwd-mode routing
-fi
 
-$kubectl get deployment/kube-dns -n kube-system -o json >kube-dns.yaml
-$kubectl delete deployment/kube-dns -n kube-system
+sleep 10
+for i in {0..10}; do
```

Comment:
I am pretty sure on a new install this will take longer, especially if the network speed is low (for the docker pulls).
Can you please make it something like 60 (wait for 5 mins)